src/SFML/Window/GlContext.cpp

```diff
@@ -446,6 +471,25 @@ GlContext* GlContext::create(const ContextSettings& settings, unsigned int width
 
     Lock lock(mutex);
 
+    // If resourceCount is 1 we know that we are inside sf::Context or sf::Window
```

Comment:
DRY code: re-factor this code and the identical one from above into a function. Maybe a template function to accommodate for the width/height vs owner/bpp difference? If the logic is the same (except how `context` is constructed), I think it makes sense.

Reply:
I initially tried to do this, but `GlContext::initialize()` is private.